test/widgets/action_sheet_test.dart

```diff
@@ -388,10 +388,16 @@ void main() {
     });
 
     testWidgets('not offered in MentionsNarrow (composing to reply is not yet supported)', (tester) async {
-      final message = eg.streamMessage();
+      final message = eg.streamMessage(flags: [MessageFlag.mentioned]);
       await setupToMessageActionSheet(tester, message: message, narrow: const MentionsNarrow());
```

Comment:
action_sheet test: Create message with the appropriate flag.
    
    The message wouldn't otherwise appear in MentionsNarrow.

Hmm indeed. But I'm a bit puzzled then: why wasn't this test already failing?

Ah it's because the code actually *will* show the message. So this is just a realism fix, making the simulated server response something a non-buggy server might give on this request.